profiles/seccomp/seccomp_default.go

```diff
@@ -356,6 +356,13 @@ func DefaultProfile() *types.Seccomp {
 			Action: types.ActAllow,
 			Args:   []*types.Arg{},
 		},
+		{
+			Names:  []string{"ptrace"},
+			Action: types.ActAllow,
+			Includes: types.Filter{
+				MinKernel: "4.8.0",
```

Comment:
Technically the kernel version is `4.8`, there is not a `4.8.0` release, ir goes `4.8`, `4.8.1`...